### ISSUE TITLE
Some environments don't support stream.clearline

### ISSUE THREAD

**Issue by mike-ward** [clessg/progress-bar-webpack-plugin#1]
I get this error when I run the Publish in Visual Studio 2015. It would be better if the plugin suppressed the error and fallback to some other behavior (displaying no progress would be fine).

```
> TBMIS@1.0.0 build C:\Users\Mike\Documents\GitHub\amgen-tbmis\amgen-tbmis\js
1>  > webpack -p
1>  
1>  C:\Users\Mike\Documents\GitHub\amgen-tbmis\amgen-tbmis\js\node_modules\progress\lib\node-progress.js:177
1>      this.stream.clearLine();
1>                  ^
1>  
1>  TypeError: this.stream.clearLine is not a function
1>      at ProgressBar.terminate (C:\Users\Mike\Documents\GitHub\amgen-tbmis\amgen-tbmis\js\node_modules\progress\lib\node-progress.js:177:17)
1>      at ProgressBar.tick (C:\Users\Mike\Documents\GitHub\amgen-tbmis\amgen-tbmis\js\node_modules\progress\lib\node-progress.js:91:10)
1>      at ProgressBar.update (C:\Users\Mike\Documents\GitHub\amgen-tbmis\amgen-tbmis\js\node_modules\progress\lib\node-progress.js:166:8)
1>      at C:\Users\Mike\Documents\GitHub\amgen-tbmis\amgen-tbmis\js\node_modules\progress-bar-webpack-plugin\index.js:30:9
1>      at Compiler.<anonymous> (C:\Users\Mike\Documents\GitHub\amgen-tbmis\amgen-tbmis\js\node_modules\webpack\lib\ProgressPlugin.js:77:4)
1>      at Compiler.applyPlugins (C:\Users\Mike\Documents\GitHub\amgen-tbmis\amgen-tbmis\js\node_modules\tapable\lib\Tapable.js:26:37)
1>      at Compiler.<anonymous> (C:\Users\Mike\Documents\GitHub\amgen-tbmis\amgen-tbmis\js\node_modules\webpack\lib\Compiler.js:193:12)
1>      at Compiler.emitRecords (C:\Users\Mike\Documents\GitHub\amgen-tbmis\amgen-tbmis\js\node_modules\webpack\lib\Compiler.js:282:37)
1>      at Compiler.<anonymous> (C:\Users\Mike\Documents\GitHub\amgen-tbmis\amgen-tbmis\js\node_modules\webpack\lib\Compiler.js:187:11)
1>      at C:\Users\Mike\Documents\GitHub\amgen-tbmis\amgen-tbmis\js\node_modules\webpack\lib\Compiler.js:275:11
1>      at Compiler.applyPluginsAsync (C:\Users\Mike\Documents\GitHub\amgen-tbmis\amgen-tbmis\js\node_modules\tapable\lib\Tapable.js:60:69)
1>      at Compiler.afterEmit (C:\Users\Mike\Documents\GitHub\amgen-tbmis\amgen-tbmis\js\node_modules\webpack\lib\Compiler.js:272:8)
1>      at Compiler.<anonymous> (C:\Users\Mike\Documents\GitHub\amgen-tbmis\amgen-tbmis\js\node_modules\webpack\lib\Compiler.js:267:14)
1>      at C:\Users\Mike\Documents\GitHub\amgen-tbmis\amgen-tbmis\js\node_modules\webpack\node_modules\async\lib\async.js:52:16
1>      at done (C:\Users\Mike\Documents\GitHub\amgen-tbmis\amgen-tbmis\js\node_modules\webpack\node_modules\async\lib\async.js:248:21)
1>      at C:\Users\Mike\Documents\GitHub\amgen-tbmis\amgen-tbmis\js\node_modules\webpack\node_modules\async\lib\async.js:44:16
1>      at FSReqWrap.oncomplete (fs.js:82:1
```

**Comment by clessg** [comment]
Hmm, I wonder if it's because it's not being run in a TTY context. I just published 1.1.0, please see if that works.

**Comment by mike-ward** [comment]
Working in 1.1.0. Thanks!
